packages/run-protocol/test/amm/vpool-xyk-amm/test-liquidity.js

```diff
@@ -471,7 +471,7 @@ test('add wrong liquidity', async t => {
   );
 
   await t.throwsAsync(() => E(addLiquiditySeatBreaking).getOfferResult(), {
-    message: /liquidity brand must be \[object Alleged: MoolaLiquidity brand]/,
+    message: /liquidity brand must be "\[Alleged: MoolaLiquidity brand]"/,
```

Comment:
```suggestion
    message: /liquidity brand must be "\[Alleged: MoolaLiquidity brand\]"/,
```
I find it bizarre that the close square bracket did not need to be escaped.

Reply:
JS R.E.s are consistent. If there's no open square bracket, close square bracket doesn't need to be escaped.

This is from experiment. I haven't found a place where the docs admit this.

Reply:
I would still prefer to see the backslash before the close square bracket. It is at least as correct, and its correctness depends only on more local information.